APMrover2/mode_breadcrumb.cpp

```diff
@@ -0,0 +1,183 @@
+#include "mode.h"
+#include "Rover.h"
+
+
+bool ModeBreadcrumb::_enter()
+{
+    //don't call enter again if _crumbs exists
+    if (_crumbs != nullptr) {
+        return false;
```

Comment:
As mentioned you do need to call the guided mode _enter, even if you don't have stuff you want to do.  Suggest inverting this check (`if (_crumbs == nullptr) { ... do our own initialsation}`)